src/building/suggested.md

```diff
@@ -3,6 +3,19 @@
 The full bootstrapping process takes quite a while. Here are some suggestions
 to make your life easier.
 
+## Installing a `pre-commit` hook
+
+The `rust` CI will automatically fail your build if it doesn't pass `tidy`, our
```

Comment:
tiny nit: I think this should either be `rust-lang/rust` or just 'Rust'.

Reply:
How about just "CI"? ;)

```suggestion
CI will automatically fail your build if it doesn't pass `tidy`, our
```